### ISSUE TITLE
JSON .csslintrc breaks when given more than one option

### ISSUE THREAD

**Issue by jawshooah** [CSSLint/csslint#567]
Given `.csslintrc`
```json
{
  "ignore": ["important"]
}
```
and `a.css`:
```css
.a {
  color: red !important;
}
```
we get the expected output:
```bash
$ csslint --config=.csslintrc a.css

csslint: No errors in a.css.
```
However, if we add another element to `.csslintrc`:
```json
{
  "ignore": ["important"],
  "format": ["text"]
}
```
We get something else:
```bash
$ csslint --config=.csslintrc a.css

csslint: There is 1 problem in a.css.

a.css
1: warning at line 2, col 3
Use of !important
  color: red !important;
```
Inspecting the source code, we can find the root of the problem in `src/cli/common.js`, specifically in `readConfigData()`:
```js
for (var optionName in json) {
    if (json.hasOwnProperty(optionName)) {
        data += "--" + optionName + "=" + json[optionName].join();
    }
}
...
options = processArguments(data.split(/[\s\n\r]+/m));
```
No whitespace is ever used to delimit arguments taken from `json`, so the array passed to `processArguments` will only ever have at most one element. In the case above, that element was `"--ignore=important--format=text"`, which obviously doesn't mean anything.

I'll submit a PR to fix this in a bit. Just wanted to log this when I saw it.

**Comment by tsmiley3** [comment]
I'm in the very early stages of learning on Android.  All the crazy coding,  syntax errors and out of place colors you see aren't there on purpose or being ignored.  I just don't know how to fix them. Thank for helping.  I need all the help I can get.          From Terri Smiley

--- Original Message ---
